src/api/server/sync_io.rs

```diff
@@ -742,8 +776,21 @@ impl<F: FileSystem + Sync> Server<F> {
                     out.max_pages = MAX_REQ_PAGES;
                     out.max_write = MAX_REQ_PAGES as u32 * pagesize() as u32; // 1MB
                 }
-                let vers = ServerVersion { major, minor };
-                self.vers.store(Arc::new(vers));
+
+                #[cfg(target_os = "linux")]
+                let meta = ServerMeta {
+                    version: ServerVersion { major, minor },
+                    support_splice_read: capable.contains(FsOptions::SPLICE_READ),
+                    support_splice_write: capable
+                        .contains(FsOptions::SPLICE_WRITE | FsOptions::SPLICE_MOVE),
```

Comment:
I have a question here(I'm not sure), should it be 
```
support_splice_write: capable
                        .contains(FsOptions::SPLICE_WRITE | FsOptions::SPLICE_MOVE),
```

or 

```
support_splice_write: capable
                        .contains(FsOptions::SPLICE_WRITE) || capable.contains(FsOptions::SPLICE_MOVE),
```
?

Reply:
There is no case that kernel fuse only support one of them. 
So first way is for confirming that all feature flags were ready.

Reply:
Got it.